app/models/concerns/orchestration.rb

```diff
@@ -170,20 +170,19 @@ def process(queue_name)
 
     rollback
   ensure
-    unless q.nil?
-      if processed > 0
-        logger.info("Processed #{processed} tasks from queue '#{q.name}', completed #{q.completed.count}/#{q.all.count}") unless q.empty?
-        # rubocop:disable Rails/FindEach
-        q.all.each do |task|
-          msg = "Task '#{task.name}' *#{task.status}*"
-          if task.status?(:completed) || task.status?(:pending)
-            logger.debug msg
-          else
-            logger.error msg
-          end
+    # rubocop:enable Rails/FindEach
+    if !q.nil? && (processed > 0)
+      logger.info("Processed #{processed} tasks from queue '#{q.name}', completed #{q.completed.count}/#{q.all.count}") unless q.empty?
+      # rubocop:disable Rails/FindEach
+      q.all.each do |task|
```

Comment:
I think you can also disable it with just this. Then you can avoid the other control statements.
```suggestion
      q.all.each do |task|# rubocop:disable Rails/FindEach
```